### ISSUE TITLE
converted a react native component from js to ts where it should be tsx

### ISSUE THREAD

**Issue by ammaraltahan** [gregjacobs/js-to-ts-converter#15]
In reference to #12 

I've noticed a couple of components were converted from js to ts where it should be tsx, and here's the code:
````javascript
import React from "react";
import { Platform, StyleSheet } from "react-native";
import type { ColorProp } from "react-native/Libraries/StyleSheet/ColorPropType";
import { Ionicons as Ionicon } from "@expo/vector-icons";

// import colors, fonts
import Colors from "../../Theme/Colors";

// Icon Config
const ICON_COLOR = Colors.black;
const ICON_SIZE = 22;

// Icon Styles
const styles = StyleSheet.create({
  icon: {
    alignSelf: "center",
    textAlign: "center"
  }
});

// Icon Props
type Props = {
  color: ColorProp,
  name: string,
  size: number
};

// Icon
const Icon = ({ color = ICON_COLOR, name, size = ICON_SIZE }: Props) => {
  const iconSize = Platform.OS === "ios" ? size + 2 : size;

  return (
    <Ionicon
      name={name}
      color={color}
      size={iconSize}
      style={[
        {
          height: iconSize,
          width: iconSize
        },
        styles.icon
      ]}
    />
  );
};

export default Icon;
````

**Comment by gregjacobs** [comment]
Hey @ammaraltahan, thanks for letting me know! I'll look into why this might be. I have a feeling it has to do with the single self-closing jsx tag not being detected where the util may only look for opening tags

**Comment by gregjacobs** [comment]
Fixed in v0.18.1 - thanks for the report!
